source/MaterialXRender/OpenGL/GlslProgram.cpp

```diff
@@ -713,7 +715,21 @@ void GlslProgram::bindLighting(HwLightHandlerPtr lightHandler, ImageHandlerPtr i
             }
         }
 
-        // Set all parameters
+        // Set all inputs
```

Comment:
No inputs were being read here. We moved from the old interface of an intermediate class with a "get parameters" interface to directly reading the node from a doc. All of our lights are defined as inputs (not parameters) -- hence no binding.